### PR TITLE
Setting to clear current clipboard when clearing history #392

### PR DESCRIPTION
**Pull request by andrewpeal**

### CODE REVIEW

**Comment by andrewpeal** [comment]
This is a PR to implement issue #392. Obviously with newer versions of Xcode lots of constraints are broken in IB, but I worked within the current framework. GUI implemented in Advanced Settings, IB connections created, UserDefaults extended to include additional property, and functionality implemented clearing NSPasteboard. FB appreciated.

File: Maccy/Maccy.swift [hunk 1]
```diff
@@ -118,6 +119,9 @@ class Maccy: NSObject {
     withClearAlert(suppressClearAlert: suppressClearAlert) {
       self.history.clearUnpinned()
       self.menu.clearUnpinned()
+      if (UserDefaults.standard.clearSystemClipboard) {
```
[the remark this answers is not in the document]
Comment (andrewpeal) [reply Maccy/Maccy.swift]: Good idea. Agreed, this is now done.

File: Maccy/Preferences/Base.lproj/AdvancedPreferenceViewController.xib [hunk 2]
```diff
@@ -131,14 +135,41 @@ defaults write org.p0deje.Maccy ignoreEvents false</string>
...
+                                </buttonCell>
+                                <connections>
+                                    <action selector="clearSystemClipboardChanged:" target="-2" id="1Lb-Tj-REC"/>
+                                </connections>
+                            </button>
+                        </gridCell>
+                        <gridCell row="QpM-uB-0J5" column="8aH-p4-Hiw" id="yrJ-wa-dsY">
+                            <textField key="contentView" verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="o6L-Xg-UBJ">
+                                <rect key="frame" x="-2" y="0.0" width="402" height="28"/>
+                                <textFieldCell key="cell" selectable="YES" title="When enabled, clearing the history would also clear the current system clipboard." id="SJh-zS-Osw">
```
[the remark this answers is not in the document]
Comment (andrewpeal) [reply Maccy/Preferences/Base.lproj/AdvancedPreferenceViewController.xib:166]: Language support has now been updated for all supported languages.

**Comment by andrewpeal** [comment]
Apologies for the CI violation, attempting to locate the reason for the failure.

**Comment by p0deje** [comment]
I hope it's okay - I've taken the liberty to clear some things from this PR, fix swiftlint issue, squash and merge via e0e775c53ccdbd6d5f87849a302feb44a51ca7d5. Thank you!

**Pull request closed without merging**